### PR TITLE
suites: thrash old client missing SLOW_OPS in the ignore-list

### PR DESCRIPTION
**Pull request by NitzanMordhai**
The current ignore-list have \(SLOW_OPS\)  but missing SLOW_OPS

Fixes: https://tracker.ceph.com/issues/66604

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: qa/tasks/thrashosds-health.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ overrides:
       - \(SMALLER_PGP_NUM\)
       - \(OBJECT_
       - \(SLOW_OPS\)
+      - SLOW_OPS
```
Comment (batrick) [inline qa/tasks/thrashosds-health.yaml:17]:
```suggestion
      - SLOW_OPS
```

?

**Review by batrick — CHANGES_REQUESTED** [review]

File: qa/tasks/thrashosds-health.yaml (hunk 1 above)
Comment (NitzanMordhai) [reply qa/tasks/thrashosds-health.yaml:17]: We also have a warning message of  - SLOW_OPS without brackets, I added it (I don't know why it shows like I also removed on line, checking

Comment (NitzanMordhai) [reply qa/tasks/thrashosds-health.yaml:17]: oh, right

**Review by batrick — APPROVED** [review]

**Comment by batrick** [comment]
jenkins test make check

**Comment by ljflores** [comment]
Looks good based on checking the thrash-old-clients tests on https://pulpito.ceph.com/yuriw-2024-07-16_01:05:51-rados-wip-yuri6-testing-2024-07-15-1335-distro-default-smithi/

No SLOW_OPS warnings.

**Comment by ljflores** [comment]
Rados approved

**Pull request merged**